db/migrate/20190203180359_create_featured_tags.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFeaturedTags < ActiveRecord::Migration[5.2]
+  def change
+    create_table :featured_tags do |t|
+      t.references :account, foreign_key: { on_delete: :cascade }
+      t.references :tag, foreign_key: { on_delete: :cascade }
```

Comment:
These references should probably be `null: false`, yes?

Reply:
I don't believe any of them are. There is no appropriate default non-null value here.